extensions/machine-learning-services/src/views/models/azureModelsComponent.ts

```diff
@@ -63,6 +66,20 @@ export class AzureModelsComponent extends ModelViewBase implements IDataComponen
 	}
 
 	public addComponents(formBuilder: azdata.FormBuilder) {
+		this.removeComponents(formBuilder);
+		if (this.azureFilterComponent && this.azureFilterComponent.data && this.azureFilterComponent.data.account) {
```

Comment:
any reason not to use null conditionals?

`if (this.azureFilterComponent?.data?.account) { ... }`

Reply:
will change. sometimes I forget that trick 


---
In reply to: [413419405](https://github.com/microsoft/azuredatastudio/pull/10125#discussion_r413419405) [](ancestors = 413419405)